apps/api/src/exception-filter.ts

```diff
@@ -0,0 +1,87 @@
+import { ArgumentsHost, Catch, ExceptionFilter, HttpException, HttpStatus } from '@nestjs/common';
+import { Response } from 'express';
+import { randomBytes } from 'crypto';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use the `node:` import directive for built-in modules to avoid the risk of colliding with modules in `node_modules`. You can read more about this directive [here](https://2ality.com/2021/12/node-protocol-imports.html).‏
```suggestion
import { randomBytes } from 'node:crypto';
```